### DISCUSSION TITLE
New User

### DISCUSSION THREAD

**Discussion by bridei** [Budibase/budibase#4657] [Help]
Hello
When a new user enter in the platform, he see something like:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/96171743/155307490-05ea7d2f-0ab2-4634-ae0c-d08a1d070abc.png">

Is it possible to:
- configure this page?
- Setup a default APP for all user to have access?
- set up a dialog to send a mail to the administrator to provide access to the desired tool?

Thanks
BR

**Comment by mjashanks** [comment]
Hey @bridei thanks for your suggestions - none of these are possible right now, but all are great suggestions. 

I would definitely like to see these be implemented, but can't give you any dates right now.
